.circleci/config.yml

```diff
@@ -170,20 +165,18 @@ jobs:
     executor: bionic
     steps:
       - checkout
-      - run: apt-get install -q -y curl gnupg
+      - run: |
+          sudo apt-get update -q
+          sudo apt-get install -q -y curl gnupg
       - run: curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor > bazel.gpg
-      - run: mv bazel.gpg /etc/apt/trusted.gpg.d/
-      - run: echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list
-      - run:
-          name: install pip
-          command: |
-            apt-get update -q
-            apt-get install -q -y python3-pip
+      - run: sudo mv bazel.gpg /etc/apt/trusted.gpg.d/
+      - run: echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | sudo tee /etc/apt/sources.list.d/bazel.list
```

Comment:
`sudo` seems odd here... but if it works...

Reply:
This is installing a new system-wide settings so it makes sense.

In general I think its better that the admin commands require sudo.. since it means that other are not running as root but a normal user.